tee-worker/omni-executor/aa-contracts/src/accounts/SmartAccount.sol

```diff
@@ -86,8 +86,8 @@ contract SmartAccount is BaseAccount, TokenCallbackHandler, UUPSUpgradeable, Ini
      * convert sender to oa bytes
      */
     function _determineOa(address sender) internal view returns (bytes32) {
-        bytes memory oaType = bytes("evm");
-        return sha256(abi.encodePacked(oaType, clientId, sender));
+        bytes3 oaType = 0x65766d;
```

Comment:
would it make difference to `bytes("evm")`? If no then no magic number is prferred, alternatively you could leave a comment too